ERCS/erc-7878.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: 7878
+title: Bequeathable Contracts
+description: A method for contracts to support inheritance of tokens after the owner's death
+author: Wamith Mockbill (@wamith)
+discussions-to: https://ethereum-magicians.org/t/new-erc-bequeathable-tokens-a-standard-to-allow-tokens-to-be-inherited-after-the-owners-death/22755
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-02-01
+requires: 20, 165, 721, 1155
+---
+
+## Abstract
+
+This EIP proposes a standard interface for contracts to allow tokens to be inherited after the owner's death. The interface allows token owners to set up a Will and designate executors to enable the transfer of tokens to inheritors after a specified waiting period.
+
+
+## Motivation
+Crypto Tokens in general and NFTs in particular are starting to be used to tokenise real-world assets. In order for them to be adopted by the main stream finance world, there needs to be a way to inherit these tokens after the owner has passed away. Currently, there is no standardised way for token holders to pass on their digital assets in the event of their death.  
+This EIP aims to solve this problem by providing a standard interface for "bequeathable" tokens, allowing for a secure and transparent process of token inheritance.
+In designing this interface we have tried to follow the real world process of Will creation and execution.
```

Comment:
Technically this line is Rationale since it's explaining how you designed your interface, but it's really just a nitpick. You can leave it here if you like.